packages/google_maps_flutter/google_maps_flutter/README.md

```diff
@@ -170,3 +170,11 @@ class MapSampleState extends State<MapSample> {
 ```
 
 See the `example` directory for a complete sample app.
+
+## Support
+
+This package is offered via an open source license. It is not governed by the Google Maps Platform Support [Technical Support Services Guidelines](https://cloud.google.com/maps-platform/terms/tssg), the [SLA](https://cloud.google.com/maps-platform/terms/sla), or the [Deprecation Policy](https://cloud.google.com/maps-platform/terms) (however, any Google Maps Platform services used by the library remain subject to the Google Maps Platform Terms of Service).
+
+This package adheres to [semantic versioning](https://semver.org/) to indicate when backwards-incompatible changes are introduced. Accordingly, while the library is in version 0.x, backwards-incompatible changes may be introduced at any time.
+
+If you find a bug, or have a feature request, please [file an issue](https://github.com/flutter/flutter/issues) on GitHub. If you would like to get answers to technical questions from other Google Maps Platform developers, ask through one of our [developer community channels](https://developers.google.com/maps/developer-community). If you'd like to contribute, please check the [Contributing guide](https://github.com/flutter/packages/blob/main/CONTRIBUTING.md).
```

Comment:
> If you would like to get answers to technical questions from other Google Maps Platform developers, ask through one of our [developer community channels](https://developers.google.com/maps/developer-community).

In the context of this README, "our" would be the Flutter team, and this is not a Flutter team resource. This seems like a potentially useful resource to link to, it'll just need language that doesn't sound like it's affiliated with the Flutter team.